frontend/src/components/LandingPage/FeaturedCourses.jsx

```
@@ -0,0 +1,176 @@
+	{
+		id: 4,
+		title: "Competitive Programming Foundational Course",
+		subject: "Computer Science",
+		description:
+			"Start the thrilling journey of competitve programming with Competitive Programming Foundational Course where various techniques, contest hacks, DSA will be taught.",
+		difficulty: "Beginner",
+		rating: 4.7,
+	},
+];
+
+let NoOfCardsToShow = 3;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'NoOfCardsToShow' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)